commit 6dbbb4e522264bcafa572da31d1e7617d2de3c3f
Author: Kefu Chai <kchai@redhat.com>
Date:   Fri Aug 3 19:50:26 2018 +0800

    mgr: silence -Wunused-lambda-capture warning
    
    ceph/src/mgr/DaemonServer.cc:1772:17: warning: lambda capture 'host' is
    not used [-Wunused-lambda-capture]
              devid, [&f, &host] (const DeviceState& dev) {
                           ^
    
    Signed-off-by: Kefu Chai <kchai@redhat.com>

diff --git a/src/mgr/DaemonServer.cc b/src/mgr/DaemonServer.cc
index d94e663ae1e..6e0eb4a38ae 100644
--- a/src/mgr/DaemonServer.cc
+++ b/src/mgr/DaemonServer.cc
@@ -1769,7 +1769,7 @@ bool DaemonServer::handle_command(MCommand *m)
       f->open_array_section("devices");
       for (auto& devid : devids) {
 	daemon_state.with_device(
-	  devid, [&f, &host] (const DeviceState& dev) {
+	  devid, [&f] (const DeviceState& dev) {
 	    f->dump_object("device", dev);
 	  });
       }
